Graphviz/syntax_test_dot.dot

```diff
@@ -32,3 +36,7 @@ edge n { lhead="" }
 
 graph n { page=""}
 //        ^^^^ support.constant.attribute.graph.dot
+
+graph cluster_n {}
+//    ^^^^^^^^^ entity.name.other.graph.dot
```

Comment:
This should probably be `entity.name.graph`, and then update the `Symbol List.tmPreferences`.